environment.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - cspice
   - csm>=3.0.3,<3.0.4
   - curl
+  - cxx-compiler=1.1.2
```

Comment:
Can we remove this install from the jenkinsfile if we are installing it here as well?